### ISSUE TITLE
Calculated checksum is same for every artifact

### ISSUE THREAD

**Issue by domhanak** [release-engineering/offliner#44]
Hello,

I am running, the following command:
```
java -jar offliner-1.6.jar -r http://repo.maven.apache.org/maven2 list-checksum-path.txt
```
`list-checksum-path.txt` - has <checksum, path> format
After the command execution finishes, I see following output:
```
11808 downloads succeeded.
0 downloads avoided.
21591 downloads failed.
```
Examining `errors.log` file I noticed the following:
```
Checksum mismatch on file: org/springframework/spring-tx/4.3.3.RELEASE/spring-tx-4.3.3.RELEASE.pom (calculated: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'; expected: '6a90721c867779d5bba871dcc889f6b7')
```
```
java.io.IOException: Checksum mismatch on file: io/netty/netty-transport-native-unix-common/4.1.31.Final/netty-transport-native-unix-common-4.1.31.Final.pom (calculated: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'; expected: '523fe1f256291eb3be6eca96993cfcc2')
```
On many artifacts. 
Grepping for `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` in the `errors.log` file yields 6000+ occurrences. This indicates that the checksum calculated by the tool is the same for each artifact.

I managed to reproduce it on colleagues machine too.

Any ideas what the issue is or what I can do to fix this?  I might be missing something. Any help is welcome.

**Comment by ruhan1** [comment]
I figured it out. PR is submitted https://github.com/release-engineering/offliner/pull/46

**Comment by jdcasey** [comment]
I've merged this PR and pushed a new snapshot build. @domhanak can you retry with:

https://oss.sonatype.org/content/repositories/snapshots/com/redhat/red/offliner/offliner/1.7-SNAPSHOT/offliner-1.7-20190618.140659-1.jar

**Comment by domhanak** [comment]
@jdcasey I tried with 1.7 version and the issue is gone. Looks good to me.

**Comment by jdcasey** [comment]
Released as part of the 1.7 release. Closing this.
